artemis-server/src/main/java/org/apache/activemq/artemis/core/server/metrics/BrokerMetricNames.java

```diff
@@ -22,5 +22,5 @@
    public static final String CONNECTION_COUNT = "connection.count";
    public static final String TOTAL_CONNECTION_COUNT = "total.connection.count";
    public static final String ADDRESS_MEMORY_USAGE = "address.memory.usage";
-
+   public static final String DISK_STORE_USED = "disk.store.usage";
```

Comment:
Your names & descriptions alternate between "used" and "usage". You should use the same verbiage everywhere to avoid confusion. For example, this line could be `DISK_STORE_USED = "disk.store.used"` or `DISK_STORE_USAGE = "disk.store.usage"`. I would go with "usage" since the address memory metric already follows this pattern.